pkgs/tools/security/ripasso/cursive.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkgconfig, ncurses, python3, openssl, libgpgerror, gpgme, xorg }:
+
+with rustPlatform;
+buildRustPackage rec {
+
+  version = "b64b1d9b697b2246b96db24a9d9ae571eab839aa";
+  name = "ripasso-cursive-${version}";
+
+  src = fetchFromGitHub {
+    owner = "cortex";
+    repo = "ripasso";
+    sha256 = "0vc56vjlq3xspvk77d4k49z9l6hs5707har7hgij0bs95k7w9lq0";
```

Comment:
The checksum for that revision seems to be `1lzim92q26gg225bdpz5k1pk826438ghp3irp9j0x9p7k6qfwjn3`